docs/binaries/santad.md

```diff
@@ -4,42 +4,58 @@ parent: Binaries
 
 # santad
 
-The santad process does the heavy lifting when it comes to making decisions
-about binary executions. It also handles brokering all of the XPC connections
-between the various components of Santa. It does all of this with performance
-being at the forefront.
+The `santad` process does the heavy lifting when it comes to making decisions
+about binary executions, file access, and mounting USB mass storage devices. It
+also handles brokering all of the XPC connections between the various components
+of Santa. It does all of this with performance being at the forefront.
 
-##### A note on performance
+## On Launch
 
-On an idling machine, santad and the other components of Santa consume virtually
-no CPU and a minimal amount of memory (5-50MB). When lots of processes
-`execve()` at the same time, the CPU and memory usage can spike. All of the
-`execve()` decisions are made on high priority threads to ensure decisions are
-posted back to the kernel as soon as possible. A watchdog thread will log
-warnings when sustained high CPU (>20%) and memory (>250MB) usage by santad is
-detected.
+When `santad` starts several pieces are initially configured:
 
-##### On Launch
+*   The rule and event databases are initialized
+*   Connections to `Santa` (GUI) and `santasyncservice` daemon are established.
```

Comment:
The connections themselves aren't created, just the listeners for those connections.

Reply:
Updated to be more specific.